cfn/googlestackdriver_collector.template

```diff
@@ -0,0 +1,858 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Description": "Alert Logic template for creating a Googlestackdriver log collector",
+    "Parameters": {
+        "AlertlogicAccessKeyId": {
+            "Description": "Alert Logic Access Key Id obtained from AIMS",
+            "Type": "String"
+        },
+        "AlertlogicSecretKey": {
+            "Description": "Alert Logic Secret Key returned from AIMS for the Access Key Id",
+            "Type": "String",
+            "NoEcho": true
+        },
+        "AlApiEndpoint": {
+            "Description": "Alert Logic API endpoint",
+            "Type": "String",
+            "Default": "api.global-services.global.alertlogic.com",
+            "AllowedValues": [
+                "api.global-services.global.alertlogic.com",
+                "api.global-integration.product.dev.alertlogic.com"
+            ]
+        },
+        "AlDataResidency": {
+            "Description": "Alert Logic Data Residency",
+            "Type": "String",
+            "Default": "default",
+            "AllowedValues": ["default"]
+        },
+        "PackagesBucketPrefix": {
+            "Description": "S3 bucket name prefix where collector packages are located.",
+            "Type": "String",
+            "Default": "alertlogic-collectors"
+        },
+        "PawsCollectorTypeName": {
+            "Description": "A collector type name. For example, okta, auth0",
+            "Type": "String",
+            "Default": "googlestackdriver"
+        },
+        "CollectorFunctionMemorySize": {
+            "Description": "Memory size for a collector function",
+            "Type": "Number",
+            "Default": 256
+        },
+        "CollectorFunctionTimeout": {
+            "Description": "Invocation timeout for a collector function.",
+            "Type": "Number",
+            "Default" : 300
+        },
+        "AlertlogicCustomerId": {
+            "Description": "Optional, Alert Logic customer ID which collected data should be reported for. If not set customer ID is derived from AIMs tokens",
+            "Type": "String",
+            "Default": ""
+        },
+        "CollectorId": {
+            "Description": "Optional, a collector UUID if known.",
+            "Type": "String",
+            "Default": "none"
+        },
+        "PollingInterval": {
+            "Description": "Interval in seconds between two consecutive poll requests.",
+            "Type": "Number",
+            "Default": 60
+        },
+        "GoogleProjectIds":{
+            "Description": "JSON array of porject IDs from which to draw logs",
+            "Type": "string",
+            "Default": "[]"
+        },
+        "PawsSecret": {
+            "Description": "Paste the contents of your JSON credentail file here.",
```

Comment:
Just caught this typo `credentail` whilst looking at this PR, but as these templates are boilerplate this typo is most likely in all paws collector templates.